msg.go

```diff
@@ -921,7 +921,7 @@ func packStructValue(val reflect.Value, msg []byte, off int, compression map[str
 				copy(msg[off:off+len(s)], s)
 				off += len(s)
 			case `dns:"octet"`:
-				bytesTmp := make([]byte, 0)
+				bytesTmp := make([]byte, 256*4+1)
```

Comment:
And yes a constant would be better.
On 21 Jul 2015 07:37, "Michael Haro" <notifications@github.com> wrote:

> In msg.go <https://github.com/miekg/dns/pull/231#discussion_r35073437>:
>
> > @@ -921,7 +921,7 @@ func packStructValue(val reflect.Value, msg []byte, off int, compression map[str
> >  				copy(msg[off:off+len(s)], s)
> >  				off += len(s)
> >  			case `dns:"octet"`:
> > -				bytesTmp := make([]byte, 0)
> > +				bytesTmp := make([]byte, 256*4+1)
>
> I'm unclear what 256*4+1 is. I see it's used more than once in this file.
> Should it be a constant?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/miekg/dns/pull/231/files#r35073437>.
>